webapp/channels/src/components/commercial_support_modal/__snapshots__/commercial_support_modal.test.tsx.snap

```diff
@@ -84,6 +70,28 @@ We recommend that you download additional environment details about your Matterm
         mode="info"
         onDismiss={[Function]}
       />
+      <div
+        className="CommercialSupportModal__packet_contents_download"
+      >
+        <FormattedMarkdownMessage
+          defaultMessage="**Select your support packet contents to download**"
+          id="commercial_support.download_contents"
+        />
+      </div>
+      <div
+        className="CommercialSupportModal__download"
+      >
+        <a
+          className="btn btn-primary DownloadSupportPacket"
+          href="/api/v4/system/support_packet"
```

Comment:
@cpoile I'm specifically having an issue here, I tried to explain it here: https://community.mattermost.com/core/pl/fjgf4oud3fbpdp6rhsoqktdcgw

Reply:
This was avoid by the https://github.com/mattermost/mattermost/pull/26403/commits/0656b1ca04357965038cae16c0ea34c54f0cd901 cc @cpoile 😅

Reply:
Hooray!